content/spin/kv-store-api-guide.md

```diff
@@ -68,6 +69,9 @@ fn handle_request(req: Request) -> Result<Response> {
 `open` **Operation**
 - The close operation is not surfaced; it is called automatically when the store is dropped.
 
+`set_json` and `get_json` **Operation**
+- Rust applications can [store and retrieve Rust data structures](spin/rust-components#storing-data-in-the-spin-key-value-store).
```

Comment:
I have gone with "serializable Rust types", as you have suggested because the Serde documentation refers to that same nomenclature when talking about Serde's out-of-the-box functionality. Which I guess is as far as this documentation goes; just top-level Serde usage.